serviceconnector/src/createLinker/LinkerCreateStep.ts

```diff
@@ -29,4 +31,17 @@ export class LinkerCreateStep extends AzureWizardExecuteStep<ICreateLinkerContex
     public shouldExecute(context: ICreateLinkerContext): boolean {
         return !context.linker;
     }
+
+    public getSourceResourceId(context: ICreateLinkerContext): string {
```

Comment:
Doesn't really matter, but this can be private.

Reply:
![image](https://img.gifglobe.com/grabs/parksandrec/S03E15/gif/KQ1DqaBAoR7R.gif)

Reply:
+1 for rhyming ✅